pkg/TIPA.py

```
@@ -158,593 +188,546 @@ class MessageFunc():
+        return pytesseract.image_to_string(threshold, lang="eng", config="--psm 6"), threshold
+
+    def clean_text(self, text: str) -> list:
+        wordlist = text.strip().split()
+        
+        if self.debug_mode >= 1:
+            logger.info(f"{wordlist} {wordlist[len(wordlist)-1]}")
+
+        new_wordlist = []
+        for word in wordlist:
+            if len(word) > 1 and re.match(r"^[-\(\)/.,\"\'a-zA-Z0-9_]*$", word):
+                new_wordlist.append(word.strip(r"[-'”\".`@_!#$%^&*<>?/\}{~:]"))
```

Ruff (0.8.0)

Using `.strip()` with multi-character strings is misleading

(B005)